<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1836</width>
    <height>934</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOpenimage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>511</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>511</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnectWebcam">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect webcam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStopWebcam">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop webcam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBlur">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show in blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGray">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>170</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRotate90">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate 90</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRotate180">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate 180</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRotate270">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate 270</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowhist">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show histogram</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConvertToGray">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert to Gray</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBMP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BMP</string>
    </property>
    <property name="iconSize">
     <size>
      <width>9</width>
      <height>9</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnJPG">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>JPG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPNG">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PNG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>left</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="left">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>right</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="right">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>top</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="top">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>bottom</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="bottom">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCropimage">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>120</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crop Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFlipHor">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Horizontal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVer">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vertical</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnsharpen">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>sharpen filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBlurring">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>220</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>blur filter</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtBlur">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>220</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="txtBlur_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>230</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>blur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoreSmoothing">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>170</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>smooth filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSmoothing">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>170</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>smooth filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEqulization">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>enhance image quality using : histogram equlization</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage3">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>280</y>
      <width>511</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScaling">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Scaling</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWarp">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Warp</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="gamma">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGamma">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>70</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>enhance image quality using : Gramma correction</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage2_4">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gramma</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImage2_5">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>130</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="alpha">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRotate">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>120</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rotate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1836</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
